tests/pages/contribute/contribute.py

```diff
@@ -10,7 +10,7 @@
 
 class ContributePage(ContributeBasePage):
 
-    URL_TEMPLATE = '/{locale}/contribute/'
+    URL_TEMPLATE = '/es-ES/contribute/'
```

Comment:
Rather than hard code the locale here, I'd suggest making the specific test open the old page in `test_contribute.py`:

```python
@pytest.mark.nondestructive
def test_play_video(base_url, selenium):
    page = ContributePage(selenium, base_url, locale='es-ES').open()
    video = page.play_video()
    assert video.is_displayed
    video.close()
```

As far as testing the newsletter goes, it looks like the contribute page is already [included in the test](https://github.com/mozilla/bedrock/blob/master/tests/functional/newsletter/test_newsletter_embed.py#L10) for that component. So I think we can have coverage for both the old and new page this way.

I'd suggest pushing this to `run-integration-tests` before we merge just to make sure everything is working right.